IntervalTree/IntervalTree.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.2;netstandard2.0;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard1.2;netstandard2.0;netstandard2.1;net45</TargetFrameworks>
```

Comment:
Of curiosity: why adding the `netstandard2.1`?

Reply:
Good question. I have to admit that I don't know exactly what I should put here to make the library usable and compatible with as many platforms as possible. I will happily remove it again, if it is unnecessary.

Reply:
Any project that can use netstandard2.1 should be capable of using netstandard2.0 as well.

Reply:
I think you would only need to target multiple versions of the same framework (full, core) if you needed to mullti-target features which were different / not existing from one version to another. Otherwise if you can target just the lowest version for each framework.
`<TargetFrameworks>netstandard1.2;net45</TargetFrameworks>`

Reply:
See https://github.com/mbuchetics/RangeTree/commit/a56b0d752699a2f18c3b97896f0d7658e9ceef55 why netstandard2.0 was kept.